helmfile/charts/networkpolicy-service/templates/opensearch/client.yaml

```diff
@@ -0,0 +1,56 @@
+{{ if .Values.opensearch.enabled }}
+{{- if .Values.opensearch.client.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-opensearch-client
+  namespace: opensearch-system
+spec:
+  policyTypes:
+    - Ingress
+    - Egress
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/component: opensearch-client
+  ingress:
+    - from:
+        - podSelector:
```

Comment:
```suggestion
    - from:
        {{- if .Values.global.ingressUsingHostNetwork }}
        {{- range $ip := .Values.global.scNodes.ips }}
        - ipBlock:
            cidr: {{ $ip }}
        {{- end }}
        {{- else }}
        - namespaceSelector:
            matchLabels:
              kubernetes.io/metadata.name: ingress-nginx
          podSelector:
            matchLabels:
              app.kubernetes.io/component: controller
              app.kubernetes.io/instance: ingress-nginx
        {{- end }}
        - podSelector:
```

Reply:
Why would we need to have access from nginx for this? This port is only used for internal traffic if I understood correctly

Reply:
Because the client nodes will get the ingress if they are enabled.

Reply:
You are correct, however it's on port 9200 and not on 9300 so I'll add that one as well